source/includes/steps-export-filter.yaml

```diff
@@ -1,46 +1,76 @@
-title: Navigate to the :doc:`collection </collections>` you wish to export data from.
+title: Navigate to the :doc:`collection </collections>` you wish to 
+       export data from.
 level: 4
 ref: select-collection
 content: |
   You can either select the collection from the
   :doc:`Collections </collections>` tab or click the collection in the
   left-hand pane.
+
 ---
 title: "Specify a filter in the Query Bar"
 level: 4
 ref: export-specify-filter
 content: |
-  Specify a :ref:`filter <query-bar-filter>` in the :ref:`query bar <compass-query-bar>` to export only the documents which match
-  the filter.
+  Specify a :ref:`filter <query-bar-filter>` in the 
+  :ref:`query bar <compass-query-bar>` to export only the documents 
+  which match the filter.
+
 ---
-title: Click :guilabel:`Collection` in the top-level menu and select :guilabel:`Export Collection`
+title: Click :guilabel:`Collection` in the top-level menu and select 
+       :guilabel:`Export Collection`
 level: 4
 ref: export-collection
 content: |
-
-  |
-
   .. figure:: /images/compass/export-data-filter-option-select.png
      :alt: Export data select menu
 
   |compass-short| displays the following dialog:
 
   .. figure:: /images/compass/export-data-dialog-filter.png
      :alt: Export data dialog
-     :figwidth: 600px
+     :width: 60%
```

Comment:
1. Can we make this a bit bigger? Maybe 80% width? In staging it's a little blurry.

2. When the image says "Recommended" what does that mean?

Reply:
1. Upped all the screenshots to 80% to keep them consistent.

2. I'm not sure. It does always seem to say exporting with a filter is (Recommended), even when the filter is a blank find and should be the same as exporting all docs. @mmarcon, do you know why Compass recommends exporting with a filter?